src/modules/app.ts

```diff
@@ -233,7 +245,8 @@ export class AppModule extends Module<ProtocolControl> {
   ): Promise<ModuleMetadata[]> {
     const moduleTypesToGet = filterByModuleType || [
       ModuleType.NFT,
-      ModuleType.COLLECTION,
+      ModuleType.COLLECTON,
```

Comment:
same here

Reply:
this is an outdated commit, removed it already [here](https://github.com/nftlabs/nftlabs-sdk-ts/blob/1cff35348c3e177b65976c6f39ccb060632e9f9b/src/modules/app.ts#L240)